receiver/hostmetricsreceiver/hostmetrics_linux.go

```diff
@@ -0,0 +1,71 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build linux
+
+package hostmetricsreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/hostmetricsreceiver"
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+var gopsutilEnvVars = map[string]string{
+	"HOST_PROC": "/proc",
+	"HOST_SYS":  "/sys",
+	"HOST_ETC":  "/etc",
+	"HOST_VAR":  "/var",
+	"HOST_RUN":  "/run",
+	"HOST_DEV":  "/dev",
```

Comment:
`HOST_PROC_MOUNTINFO` is missing from this list, is that intention?

Reply:
Yes that's intentional. The translations allow us to use the mountpoints under `HOST_PROC` so it doesn't need the alternate `mountinfo` file.